### PR TITLE
Some edits as I walked through the tutorial

### PR DESCRIPTION
**Pull request by staxmanade**
I walked through the [React & Webpack](http://www.typescriptlang.org/docs/handbook/react-&-webpack.html) and ran into some troubles... These changes got it working again.

It seems that [Typings](http://github.com/typings/typings) has changed a bit so people walking through the tutorial will get stuck. Hope these are the right changes but it worked for me...

### CODE REVIEW

File: pages/tutorials/React & Webpack.md [hunk 1]
```diff
@@ -83,11 +83,12 @@ If you want a local copy, just run `npm install typescript`.
 Finally, we'll use Typings to grab the declaration files for React and ReactDOM:
 
 ```shell
-typings install --ambient --save react
-typings install --ambient --save react-dom
+typings install --global --save "dt~react"
+typings install --global --save "dt~react"
```
Comment (icecreammatt) [inline pages/tutorials/React & Webpack.md]: Is this supposed to be "dt~react-dom"? Otherwise it is the same thing as the line above.

Comment (staxmanade) [inline pages/tutorials/React & Webpack.md]: Oops. Yes...

**Comment by icecreammatt** [comment]
I had an issue running through these changes originally because my typings was out of date. It wasn't a very visible bug to me until I just tried running typings alone. It wouldn't work with v0.8.1 so anyone running into this in the future be sure to upgrade typings. `Update available 0.8.1 → 1.0.3`

**Comment by DanielRosenwasser** [comment]
Typings updated to 1.0 today, so all our documentation is out of date. :sweat_smile:

File: pages/tutorials/React & Webpack.md [hunk 2]
```diff
@@ -83,11 +83,12 @@ If you want a local copy, just run `npm install typescript`.
 Finally, we'll use Typings to grab the declaration files for React and ReactDOM:
 
 ```shell
-typings install --ambient --save react
-typings install --ambient --save react-dom
+typings install --global --save "dt~react"
+typings install --global --save "dt~react-dom"
```
Comment (DanielRosenwasser) [inline pages/tutorials/React & Webpack.md]: Do you really have to do this `dt~` thing?

Comment (DanielRosenwasser) [inline pages/tutorials/React & Webpack.md]: It looks like you do. =/

File: pages/tutorials/React & Webpack.md [hunk 3]
```diff
@@ -83,11 +83,12 @@ If you want a local copy, just run `npm install typescript`.
...
 ```
 
-The `--ambient` flag will tell Typings to grab any declaration files from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), a repository of community-authored `.d.ts` files.
+The `dt~` tells typings to use the [dt | DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) soruce, a repository of community-authored `.d.ts` files.
```
Comment (DanielRosenwasser) [inline pages/tutorials/React & Webpack.md]:
```
The `--global` flag, along with the `dt~` prefix tells Typings to grab any declaration files from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), a repository of community-authored `.d.ts` files.
```

**Comment by staxmanade** [comment]
@DanielRosenwasser 👍

**Comment by DanielRosenwasser** [comment]
Thanks @staxmanade!

**Comment by staxmanade** [comment]
@DanielRosenwasser what is the release process like for this? (don't yet see it online)

**Comment by DanielRosenwasser** [comment]
Basically it's a manual publishing process. I'll try to get it out by the end of the week. Sorry for the delay!

**Pull request merged**